app/services/usps_in_person_proofing/mock/fixtures.rb

```diff
@@ -83,8 +83,12 @@ def self.request_passed_proofing_secondary_id_type_results_response_ial_2
         )
       end
 
-      def self.request_expired_proofing_results_response
-        load_response_fixture('request_expired_proofing_results_response.json')
+      def self.request_expired_enhanced_ipp_results_response
```

Comment:
Confirmed- no more instances of `request_expired_proofing_results_response` found